### DISCUSSION TITLE
Vim keybindings

### DISCUSSION THREAD

**Discussion by prologic** [outline/outline#5793] [Ideas]
Hi Team 👋

Would it be possible to have basic Vim keybindings for the editor as a user preference/option?

cheers
James

**Comment by turtlebasket** [comment]
Obsidian, HedgeDoc, HackMD, etc. all include optional Vim bindings. Would appreciate this feature in Outline as well!

**Comment by wilsonhou** [comment]
bumping this - open to contributing this!
